src/main/scala/com/fulcrumgenomics/umi/ConsensusCaller.scala

```diff
@@ -141,6 +159,22 @@ class ConsensusCaller(errorRatePreLabeling:  PhredScore,
       val base = DnaBasesUpperCase(maxLlIndex)
       (base, q)
     }
+
+    /**
+      * Returns the calculated likelihoods of the four bases given the data and the model. The likelihoods
```

Comment:
Changed, although I'm not sure I agree with your assertion.  If the computation is expensive then I think I'd prefer the documentation to make it clear that there's a computation happening every time and not a simple access.